ambry-router/src/main/java/com.github.ambry.router/PutManager.java

```diff
@@ -64,6 +64,7 @@
   private final AtomicBoolean isOpen = new AtomicBoolean(true);
   private final OperationCompleteCallback operationCompleteCallback;
   private final ReadyForPollCallback readyForPollCallback;
+  private final List<String> listToFillWithIdsToDelete;
```

Comment:
`idsToDelete` should be good enough, given the params. I actually got some confusion when first saw this longer name.